os/drivers/video/video_null_driver.c

```diff
@@ -1002,6 +1107,10 @@ static int video_null_set_format(FAR void *video_private, FAR struct v4l2_format
 	}
 
 	ret = video_null_v4l2_try_format(priv, fmt, 1);
+	/* Format changed, set worker function delay accordingly */
+	if (ret > 0) {
```

Comment:
ret != OK is correct, in my opinion.

Reply:
The value 'ret' returned by `video_null_v4l2_try_format` can be an error value (negative), the number of bytes transferred during the format trial (positive) or OK (zero).
Only in the case where transfer of non-zero bytes with the new format is successful, the format of the null driver will be changed and we need to reset the worker's delay.
Hence, only checking for `ret > 0`.